### PR TITLE
chore: add Action field to Case history (backend)

### PR DESCRIPTION
**Pull request by felixcicatt**
Added the new Action column to the Case history in the backend.
Resolves PZ-2440

### CODE REVIEW

File: src/main/kotlin/net/atos/zac/app/zaken/converter/historie/RESTZaakHistoriePartialUpdateConverter.kt [hunk 1]
```diff
@@ -27,6 +37,7 @@ class RESTZaakHistoriePartialUpdateConverter @Inject constructor(
             datumTijd = auditTrail.aanmaakdatum
             door = auditTrail.gebruikersWeergave
             toelichting = auditTrail.toelichting
+            this.actie = actie
```
Comment (hsiliev) [inline src/main/kotlin/net/atos/zac/app/zaken/converter/historie/RESTZaakHistoriePartialUpdateConverter.kt:40]: Do we test this ?

Comment (felixcicatt) [reply src/main/kotlin/net/atos/zac/app/zaken/converter/historie/RESTZaakHistoriePartialUpdateConverter.kt:40]: Meant to add this as a draft PR as well. I will at the very least extend the existing integration tests.

File: src/main/java/net/atos/zac/app/audit/model/RESTHistorieActie.java
[the remark this answers is not in the document]
Comment (felixcicatt) [reply src/main/java/net/atos/zac/app/audit/model/RESTHistorieActie.java:1]: Done :-)

File: src/main/kotlin/net/atos/zac/app/zaken/converter/historie/RESTZaakHistoriePartialUpdateConverter.kt (hunk 1 above)
Comment (felixcicatt) [reply src/main/kotlin/net/atos/zac/app/zaken/converter/historie/RESTZaakHistoriePartialUpdateConverter.kt:40]: We do now :-)

**Review by hsiliev — APPROVED** [review]

**Pull request merged**